.github/workflows/portable-rustls-ci-build.yml

```diff
@@ -111,14 +116,22 @@ jobs:
         env:
           RUSTFLAGS: --cfg portable_atomic_unstable_coerce_unsized --cfg unstable_portable_atomic_arc
 
-      - name: cargo test (debug; all features)
+      - name: cargo test (debug; unstable-std-extra [should cover all non-unsafe features])
+        run: cargo test -F unstable-std-extra --all-targets
+        working-directory: rustls
+        env:
+          RUSTFLAGS: --cfg portable_atomic_unstable_coerce_unsized --cfg unstable_portable_atomic_arc
+          RUSTDOCFLAGS: --cfg portable_atomic_unstable_coerce_unsized --cfg unstable_portable_atomic_arc
+
+      - name: cargo test (debug; all features except unsafe-assume-single-core)
+        # XXX TBD ??? ???
         # QUICK WORKAROUND TO AVOID TEST ISSUE WITH RUST NIGHTLY FROM MAY 2024
-        if: ${{ !startsWith(matrix.rust, 'nightly-2024-05') }}
-        run: cargo test --all-features
+        # if: ${{ !startsWith(matrix.rust, 'nightly-2024-05') }}
```

Comment:
Looks nice build seems to remain green with this "QUICK WORKAROUND" removed. Should remove corresponding comment for this Rust nightly version from the matrix. I would also love to understand why this "QUICK WORKAROUND" is no longer needed with these updates.